### PR TITLE
DEVTOOLS: semi-auto creation of SUBTITLES.MIX for Blade Runner

### PR DESCRIPTION
**Pull request by antoniou79**
add make rules for output FON and MIX files

Create wrappers for all scripts to check for python 2.7
update module.mk to create FON file, then create the SUBTITLES.MIX file
Remove the intermediate TRE files.
**Input is expected to be in sampleInput folder**
- subtitlesFont.png: The font glyphs input image file. Needed for the font creation (SUBTLS_E.FON)
- overrideEncodingSUBLTS.txt: A configuration file for the font creation. A sample is provided in the sampleInput folder and documentation in the readme. Needed for the font creation (SUBTLS_E.FON)
- englishTranscript.xls: The excel file with the transcript. This is the file maintained online. Needed for the  SUBTITLES.MIX. 
- configureFontsTranslation.txt. A configuration file for the MIX creation. A sample is provided in the sampleInput folder and documentation in the readme. Needed for the  SUBTITLES.MIX.

### CODE REVIEW

**Comment by antoniou79** [comment]
Should work the make command:
make devtools/create_bladerunner/subtitles

The scripts require Python 2.7 (they won't run with Python 3.0 and an error message will be displayed about that).
I'm not sure if (or how) I should also add a check for missing required library modules (namely: xlrd, PIL). 

I intend to update the documentation (readme) a bit to reflect minor changes in the scripts.

**Comment by antoniou79** [comment]
I think I've finished working on this, pending review comments.

**Comment by antoniou79** [comment]
I've tested a bit with localized versions of the game, namely DE, ITA, FR, ES. 
The English subtitles for these versions work ok. For localized subtitles (in Deutsch, Italian, French, Español respectively) another font PNG would be needed; I'm thinking a charset like Windows-1252 would fit the requirements of all these translations probably.

Some notes and observations: 
- Speech that may be a placeholder in the English version (see the quotes marked as (boop) in the Excel) can be potentially used by a localized version. I've noticed this once, but it probably happens more often. For example the blimp guy's quote 60 (60-0030.AUD) is a "boop" sound in English, but there is voice-over in Italian. 
- Some quote slots that are used in the English version may be muted in other localized versions. For example, during McCoy's initial speech, when the game starts, the last line "No self-respecting human..." (99-1860.AUD) is muted and shortened in the Deutsch version.
- The localized versions seem to also have voice-overs _for the missing / removed dialogues_ as in the English one. A more thorough check would be needed, but it all seems to be there. Just mentioning this in case there's a plan to restore those -- especially since the Designers Cut mode barely does anything (or it could be another KIA option entirely).
- The "dialogue" that exist in the SFX.MIX don't seem to be localized. This is mainly the 3 audio files with developer commentary (MA04VO1A.AUD, CT01VO1A.AUD, HC01VO1A.AUD) and a Blimp Guy talking ("COLONY.AUD"), as far as I know. 

I haven't tested with the Russian version. I don't know the language at all and have no access to the supported Russian version of the game. Can anyone who owns it confirm that the version conforms to the naming convention for the files (eg are the Text Resource files named *.TRR ?). Creating a PNG for the Russian subtitles font would not be difficult though, and I also expect that the English subtitles would work as they are with the font image provided in the sampleInput folder.

**Comment by sev-** [comment]
I own the Russian version. Where to look? Inside the MIX files?

**Comment by antoniou79** [comment]
> I own the Russian version. Where to look? Inside the MIX files?

Yes, STARTUP.MIX would do.

File: devtools/module.mk [hunk 1]
```diff
@@ -14,9 +14,9 @@ DEVTOOLS := \
 	devtools/md5table$(EXEEXT) \
 	devtools/make-scumm-fontdata$(EXEEXT)
 
-include $(srcdir)/devtools/*/module.mk
+include $(srcdir)/devtools/*/module.mk $(srcdir)/devtools/*/*/module.mk
```
Comment (sev-) [inline devtools/module.mk]: Please remove this change. Rather do the following:

add module.mk to devtools/create_bladerunner, which will contain include directive. Test it and then I'll merge

**Comment by sev-** [comment]
The language files are named *.TRE, and contain Russian text in CP-1251 encoding. Apparently, English version was used as the base for the translation.

**Comment by sev-** [comment]
Thanks! Merging, and welcome to the team!

**Pull request merged**